packages/files-ui/src/Components/Modules/FileBrowsers/ShareToSharedFolderModal.tsx

```diff
@@ -209,24 +201,17 @@ interface IShareFileProps {
 
 const CopyToSharedFolderModal = ({ close, file, filePath }: IShareFileProps) => {
   const classes = useStyles()
-  const { isCreatingSharedFolder, handleCreateSharedFolder } = useCreateOrEditSharedFolder()
+  const { handleCreateSharedFolder } = useCreateOrEditSharedFolder()
   const [sharedFolderName, setSharedFolderName] = useState("")
   const { sharedFolderReaders, sharedFolderWriters, handleLookupUser, onNewUsers, usersError } = useLookupSharedFolderUser()
   const [isUsingCurrentBucket, setIsUsingCurrentBucket] = useState(true)
-  const [keepOriginalFile, setKeepOriginalFile] = useState(true)
-  const [currentStep, setCurrentStep] = useState<Step>("1_SHARED_FOLDER_SELECTION_CREATION")
+  const [keepOriginalFile, setKeepOriginalFile] = useState(false)
```

Comment:
Shouldn't we keep this true by default, to be safe?
```suggestion
  const [keepOriginalFile, setKeepOriginalFile] = useState(true)
```